go/tasks/pluginmachinery/internal/webapi/core.go

```diff
@@ -191,7 +191,7 @@ func createRemotePlugin(pluginEntry webapi.PluginEntry, c clock.Clock) core.Plug
 			}
 
 			resourceCache, err := NewResourceCache(ctx, pluginEntry.ID, p, p.GetConfig().Caching,
-				iCtx.MetricsScope().NewSubScope("cache"))
+				iCtx.MetricsScope().NewSubScope(pluginEntry.ID).NewSubScope("cache"))
```

Comment:
Not sure I agree with this... I think `iCtx.MetricsScope()` should already be scoped to the plugin...